com.ca.lsp.cobol/lsp-core-cobol-parser/src/test/java/com/ca/lsp/core/cobol/preprocessor/sub/util/impl/ReplacingServiceImplTest.java

```diff
@@ -15,214 +15,57 @@
 
 package com.ca.lsp.core.cobol.preprocessor.sub.util.impl;
 
-import com.ca.lsp.core.cobol.parser.CobolPreprocessor.*;
 import com.ca.lsp.core.cobol.preprocessor.sub.util.ReplacingService;
-import com.ca.lsp.core.cobol.preprocessor.sub.util.TokenUtils;
-import org.antlr.v4.runtime.BufferedTokenStream;
+import org.apache.commons.lang3.tuple.Pair;
 import org.junit.jupiter.api.Test;
 
 import java.util.List;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.when;
 
 /** This test checks the logic of {@link ReplacingServiceImpl} */
```

Comment:
great job to simplify the test, I would recommend a line of comment that explain what we provide as input and what is returned as output by the replacing service

Reply:
Yeah, makes sense